lib/livebook/notebook/cell.ex

```diff
@@ -18,6 +18,13 @@ defmodule Livebook.Notebook.Cell do
 
   @type indexed_output :: {non_neg_integer(), Livebook.Runtime.output()}
 
+  @type canvas_location :: %{
+          x: non_neg_integer(),
+          y: non_neg_integer(),
+          w: non_neg_integer(),
+          h: non_neg_integer()
```

Comment:
Is there even a need for resizing the output? For charts this would be really nice, but for this to happen we would need the ability to change the code cell source

Reply:
As per the other comment, outputs generally use the available width. This is especially important for outputs that use iframes, because they live outside the main layout and we position them absolutely, so without specific width they would freak out :D